source/_integrations/syslog.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to add syslog notifications to Home Assistant.
 ha_category:
   - Notifications
 ha_release: pre 0.7
+ha_iot_class: ~
```

Comment:
`Local Polling`?

Reply:
I would say `Local Push`. There is no polling happening in this integration, and it pushed out data to the local syslog.

```suggestion
ha_iot_class: Local Push
```